flask_restalchemy/resources/utils.py

```diff
@@ -86,13 +86,20 @@ def query_from_request(model, model_serializer, request):
     :param class model:
         SQLAlchemy model class representing a database resource
 
+    :param model_serializer:
+        instance of model serializer
+
     :param request:
         Flask http request data
 
-    :rtype: list
-    :return: the items for the current page
+    :param query:
+        SQLAlchemy query instance
+
+    :rtype: list or dict (pagination)
```

Comment:
Here, the correct syntax would be `:rtype: list|dict`

Reply:
OK